support-services/docker-entrypoint.d/04-upload-thesauri.sh

```diff
@@ -18,3 +18,12 @@ do
   echo ""
```

Comment:
Is this change supposed to be part of the PR? Not sure if it is necessary for the test or it came in by mistake.

Reply:
yes definitely, I added some more logging because I've noticed that sometimes the thesauri are not correctly present when running the e2e tests, and that would cause one of the very common failures.